### DISCUSSION TITLE
Exercise 4 Ques #4

### DISCUSSION THREAD

**Discussion by emilyyli** [LiNk-NY/IntroR#40] [Q&A]
![image](https://github.com/CUNYSPHCode/IntroR/assets/171090659/07794e8c-0348-44cc-8d69-37bd4d6981d3)
For ques #4, is the question just asking us to compare the mean displacement values to each other? (ex: the mean disp for straight is 3x higher than the other).

**Comment by ralythompson24** [comment]
Yep! That's all there is to it.
